packages/shared/src/components/onboarding/FeedReadyMessage.tsx

```diff
@@ -15,6 +17,11 @@ export const FeedReadyMessage = ({
   className,
 }: FeedReadyMessageProps): ReactElement => {
   const { user } = useContext(AuthContext);
+  const onboardingOptimizations = useFeature(feature.onboardingOptimizations);
```

Comment:
Adding check here and not where this component is rendered because that is the FeedContainer that is used in every feed and would enroll also users who view the feed not for the first time. 
Based on [this](https://dailydotdev.slack.com/archives/C02E2C3C13R/p1708508556884909)